backend/src/middleware/notifications/notificationsMiddleware.spec.js

```diff
@@ -154,18 +158,18 @@ describe('notifications', () => {
             await createCommentOnPostAction()
             const expected = expect.objectContaining({
               data: {
-                currentUser: {
-                  notifications: [
```

Comment:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2019-08-30T11:09:36Z" title="Friday, August 30th 2019, 1:09:36 pm +02:00">Aug 30, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -272,6 +275,9 @@ describe('notifications', () => {
         })
 
         describe('updates the post and mentions me again', () => {
+          const expectedUpdatedContent =
+            '<br>One more mention to<br><a data-mention-id="you" class="mention" href="/profile/you" target="_blank"><br>@al-capone<br></a><br>and again:<br><a data-mention-id="you" class="mention" href="/profile/you" target="_blank"><br>@al-capone<br></a><br>and again<br><a data-mention-id="you" class="mention" href="/profile/you" target="_blank"><br>@al-capone<br></a><br>'
+
```


I have wired experiences with „variables“ directly in a describe block and not in an `before` or `it`. Just to let you know …

Reply:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-08-30T12:04:50Z" title="Friday, August 30th 2019, 2:04:50 pm +02:00">Aug 30, 2019</time>_

---


weird experiences @Tirokk?
I think you were saying something about that the other day...
it should be ok, but yeah I started adding mocks, etc in `beforeEach` blocks after I had some flaky tests